build-nginx.sh

```diff
@@ -30,7 +26,9 @@ sed -E "s/${IMAGE_ORIGINAL_TAG}/${IMAGE_TAG}/g" "Dockerfile-${IMAGE}" | docker b
     --build-arg=NGINX_VHOST_TEMPLATE=php-fpm --target="${IMAGE}-dev" -f - . \
     && echo "$USABILLA_TAG_DEV" >> "${TAG_FILE}"
 
-if [[ -n ${IMAGE_EXTRA_TAG} ]]; then
-    docker tag "${USABILLA_TAG}" "${USABILLA_EXTRA_TAG}" && echo "${USABILLA_EXTRA_TAG}" >> "${TAG_FILE}"
-    docker tag "${USABILLA_TAG_DEV}" "${USABILLA_EXTRA_TAG_DEV}" && echo "${USABILLA_EXTRA_TAG_DEV}" >> "${TAG_FILE}"
-fi
+for IMAGE_EXTRA_TAG in "${@:2}"
```

Comment:
to use multiple tags should we do now something like this:
`./build-nginx.sh 1.15 nginx nginx1`?

Reply:
Yeah, I was waiting to reabase and do that on the 1.17, good catch!